ios/app/brave_core_main.h

```diff
@@ -80,6 +81,10 @@ OBJC_EXPORT
 
 @property(readonly) AdblockService* adblockService;
 
+#if BUILDFLAG(IPFS_ENABLED)
```

Comment:
This buildflag will have no effect because this is a public header. When its included by iOS there are no buildflags defined anywhere.  If we want it to only be available when the buildflag is enabled then we have to make the property `nullable` and return nil on the `ipfsAPI` method when the buildflag is not enabled

Reply:
Actually we are going to delete this buildflag in the future, so i can just remove this ATM